torch/optim/swa_utils.py

```diff
@@ -26,6 +26,8 @@ class AveragedModel(Module):
             :class:`AveragedModel` parameter, the current value of :attr:`model`
             parameter and the number of models already averaged; if None,
             equally weighted average is used (default: None)
+        mode (str, optional): whether to use parameters or state_dict for update
```

Comment:
This should document what are the valid values for this argument.

Reply:
parameters and state_dict are valid values here as stated in line 29. Although, anything passed other than state_dict defaults to parameters.

Reply:
Ho yes, what I meant is that this is missing `"` to make it clear that these are strings.
It's a nit though.